OpenEXR/IlmImf/ImfMultiPartInputFile.cpp

```diff
@@ -639,13 +639,18 @@ MultiPartInputFile::Data::chunkOffsetReconstruction(OPENEXR_IMF_INTERNAL_NAMESPA
                 int y_coordinate;
                 OPENEXR_IMF_INTERNAL_NAMESPACE::Xdr::read <OPENEXR_IMF_INTERNAL_NAMESPACE::StreamIO> (is, y_coordinate);
                 
+                
+                if(y_coordinate < header.dataWindow().min.y || y_coordinate > header.dataWindow().max.y)
+                {
+                    // bail to exception catcher: y out of range. Test now to prevent overflow in following arithmetic
+                   throw int();
```

Comment:
throw int() occurs four times in this file, and nowhere else. I suspect it's legacy. Probably 
````
throw IEX_NAMESPACE::IoExc("y out of range");
````
and similar should replace all the throw int() in this file. Should we hit all four in this commit?

Reply:
I'll do that. I think I originally wrote those intending to refactor them out.
I think they can be replaced with `break` but throwing an exception with a message as you suggest will make it easier to debug